editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -217,6 +218,49 @@ fn vec_f32_input(document_node: &DocumentNode, node_id: NodeId, index: usize, na
 	}
 	widgets
 }
+
+fn vec_dvec2_input(document_node: &DocumentNode, node_id: NodeId, index: usize, name: &str, text_props: TextInput, blank_assist: bool) -> Vec<WidgetHolder> {
+	let mut widgets = start_widgets(document_node, node_id, index, name, FrontendGraphDataType::Color, blank_assist);
+
+	let from_string = |string: &str| {
+		let string = &Regex::new(r",+").unwrap().replace_all(string, ",").to_string();
+		string
+			.trim()
+			.trim_end_matches(')')
+			.trim_start_matches('(')
+			.trim()
+			.split("),")
+			.filter(|x| !x.is_empty())
+			.map(|item| {
+				let parts: Vec<_> = item.trim().split(",").filter(|x| !x.is_empty()).collect();
+				if parts.len() == 2 {
+					let x = parts[0].trim().trim_end_matches(')').trim_start_matches('(').trim().parse::<f64>().ok()?;
+					let y = parts[1].trim().trim_end_matches(')').trim_start_matches('(').trim().parse::<f64>().ok()?;
+					Some(DVec2 { x: x, y: y })
+				} else {
+					None
+				}
+			})
+			.collect::<Option<Vec<_>>>()
+			.map(TaggedValue::VecDVec2)
```

Comment:
If we want to be more forgiving with the syntax, allowing the style of `<path d=` in SVG, we could do something like:
```rs
string
	.split(|c: char| !c.is_alphanumeric() && !matches!(c, '.' | '+' | '-'))
	.filter(|x| !x.is_empty())
	.map(|x| x.parse::<f64>().ok())
	.collect::<Option<Vec<_>>>()
	.map(|numbers| numbers.chunks_exact(2).map(|vals| DVec2::new(vals[0], vals[1])).collect())
	.map(TaggedValue::VecDVec2);
```